SAVideoRangeSlider/SAVideoRangeSlider.m

```diff
@@ -330,14 +332,15 @@ -(void)getMovieFrame{
     
     AVAsset *myAsset = [[AVURLAsset alloc] initWithURL:_videoUrl options:nil];
     self.imageGenerator = [AVAssetImageGenerator assetImageGeneratorWithAsset:myAsset];
+    self.imageGenerator.appliesPreferredTrackTransform = YES;
```

Comment:
apply the track matrix (or matrices) when extracting an image from the asset.
correct orientation while extracting the frame image.
Cooperate with the maximumSize of AVAssetImageGenerator, you did not adjust the width of frameImageView` frame.

Remove the code like this:
CGRect rect=tmp.frame;
rect.size.width=picWidth;
tmp.frame=rect;